plugins/embark-slither/src/index.ts

```diff
@@ -0,0 +1,40 @@
+import fs from "fs-extra";
+import shelljs from "shelljs";
+
+function isSlitherInstalled() {
+  return shelljs.which("slither");
+}
+
+function executeSlither(path: string) {
+  shelljs.exec(`slither ${path}`);
+}
+
+function getHeader(path: string) {
+  return `
+JSON AST (compact format):
+
+
+======= ${path} =======
+
+`;
+}
+
+function run(compilationResult: { sources: { [path: string]: { ast: object }}}) {
+  if (!isSlitherInstalled()) {
+    console.log("Slither is not installed, visit: https://github.com/trailofbits/slither");
+    return;
+  }
+
+  Object.keys(compilationResult.sources).forEach((path) => {
+    const ast = compilationResult.sources[path].ast;
+    const astPath = `${path}.ast.json`;
```

Comment:
No substitutions are needed to avoid have `/` or `\` in the filename? I think Windows might complain about this.